### DISCUSSION TITLE
Alternative for buildStoriesJson ?

### DISCUSSION THREAD

**Discussion by iamtheozzy** [storybookjs/storybook#26964] [Help]
### Summary

I'm noticing that the `buildStoriesJson` option has been removed from the [features option](https://storybook.js.org/docs/api/main-config-features#legacydecoratorfileorder) of Storybook v8. I'm wondering if anyone knows of an alternative or how to get a similar result.

**Comment by StefDev** [comment]
There is a `index.json` file (also in the _public_ folder) that may contain the data you need.
